examples/function-pointers/function_pointers.bal

```diff
@@ -2,15 +2,20 @@ import ballerina/io;
 
 // The 'test' function acts as a variable function pointer in the 'main' function
 function test(int x, string s) returns (float) {
-    int y = check <int> s;
-    float f = x * 1.0 * y;
+    int|error y = int.create(s);
+    float f = 0.0;
+    if (y is int) {
+        f = x * 1.0 * y;
+    } else if (y is error) {
+        panic y;
+    }
     return f;
 }
 
-// Function pointer as a parameter.
+// Function pointer as a parameter. And use `call` function to invoke the function using the function pointer.
```

Comment:
Remove "And". Also, shall we say "function foo" to be more specific.

Reply:
Removed "And". Change will come with the next commit. Did not use specific language because code is susceptible to change frequent. Thus used the comment to explain the concept than the direct code. This is frequent in many other places.